### PR TITLE
Michael/getting started

### PR DESCRIPTION
**Pull request by michaelpkuhn**
- Expands README
- Adds set up scripts
- Adds example notebook

TODO:
- This relies on the current implementation of the Census data
  - Plan is to refactor functions into another script
  - The getting started notebook will likely need modification upon this change

### CODE REVIEW

File: tests/test_environment.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from dotenv import load_dotenv
+import os
+import sys
+
+load_dotenv()
+
+sys.path.append(os.path.abspath(''))
```
Comment (brickman1444) [inline tests/test_environment.py:7]: Can you add a comment explaining why this is needed?

File: getting_started/example.ipynb [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "c:\\Users\\kuhnie\\code-chicago\\gcfd\\code-for-chicago-gcfd\n"
```
Comment (brickman1444) [inline getting_started/example.ipynb:38]: Does this matter? Looks like it's a local path on your machine

File: getting_started/example.ipynb [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+      "        View dataframes: df_dict[key] (key='zip' or 'county')\n",
+      "    Save dataframe using CensusData.df_to_json()\n",
+      "        Default saves zipped by geo_code\n",
+      "        Set zip_df = False to save dataframes without processing\n",
+      "    Load dataframe using CensusData.load_df()\n",
+      "        Default loads unzipped saved file, described above\n",
+      "\u001b[1;31mInit docstring:\u001b[0m\n",
+      "Initialized instance\n",
+      "Adds metric to class set data_metrics\n",
+      "\u001b[1;31mFile:\u001b[0m           c:\\users\\kuhnie\\code-chicago\\gcfd\\code-for-chicago-gcfd\\src\\census_response.py\n",
```
Comment (brickman1444) [inline getting_started/example.ipynb:216]: local path here too

File: README.md [hunk 4]
```diff
@@ -5,26 +5,35 @@
...
+       - Windows: `python src\main.py`
+     - To run with memory profiling on MacOS/Linux:
+         1. Make sure the script is executable: `chmod 755 memory_profiling/run_main.sh`
+         2. Run the script: `./memory_profiling/run_main.sh`
+         3. View the report: `memory_profiling/memory_profile_report.txt`
+     - To run with memory profiling on Windows CMD/Powershell:
+         1. Run the script: `.\memory_profiling\run_main.bat`
+         2. View the report: `memory_profiling\memory_profile_report.txt`
+ 1. Explore Data:
+      - After code runs, then you can open the dataframes in `getting_started/example.ipynb`
```
Comment (brickman1444) [inline README.md:36]: Can you add how to open the .ipynb file? As someone unfamiliar with Jupiter Notebook I wouldn't even know how to open it.

**Review by brickman1444 — CHANGES_REQUESTED** [review]
The readme and setup changes look good.

I'm not familiar with how Jupiter Notebook works. Does example.ipynb actually contain data or does it just point to data somewhere else?

**Comment by michaelpkuhn** [comment]
> The readme and setup changes look good.
> 
> I'm not familiar with how Jupiter Notebook works. Does example.ipynb actually contain data or does it just point to data somewhere else?

It runs code in cells. Github actually displays the format pretty well on the [file's page](https://github.com/Code-For-Chicago/greater-chicago-food-despository/blob/michael/getting-started/getting_started/example.ipynb)

File: getting_started/example.ipynb (hunk 2 above)
Comment (michaelpkuhn) [reply getting_started/example.ipynb:38]: No, jupyter notebooks store that data. There might be a way to clear that

File: README.md (hunk 4 above)
Comment (michaelpkuhn) [reply README.md:36]: You can open it in VS code with the Python and Jupyter extensions.

Comment (michaelpkuhn) [reply README.md:36]: If you don't use VS code, you could also open with [JupyterLab](https://jupyter.org/install)

I'll add to readme

File: tests/test_environment.py (hunk 1 above)
Comment (michaelpkuhn) [reply tests/test_environment.py]: Works without it, not sure why I added it.

**Review by brickman1444 — APPROVED** [review]

**Pull request merged**